suzieq/cli/sqcmds/command.py

```diff
@@ -203,11 +237,11 @@ def describe(self):
         self.ctxt.exec_time = "{:5.4f}s".format(time.time() - now)
         return self._gen_output(df)
 
-    @command("top", help="find the top n values for a field")
-    @argument("count", description="number of rows to return")
-    @argument("what", description="integer field to get top values for")
-    @argument("reverse", description="return bottom n values",
-              choices=['True', 'False'])
+    @ command("top", help="find the top n values for a field")
+    @ argument("count", description="number of rows to return")
+    @ argument("what", description="numeric field to get top values for")
+    @ argument("reverse", description="return bottom n values",
```

Comment:
Only visual comment. These spaces between `@` and the command shouldn't exists

Reply:
I agree. That thing is added by autopep8 or something, I have to see which one. I don't add the space when I write the code